oauth-overview.html.md.erb

```diff
@@ -0,0 +1,106 @@
+
+* [Authorization Code (3-legged) Tutorial](./oauth-authorisation-code.html)
+* [Client Credentials (2-legged) Tutorial](./oauth-client-credentials.html)
+* [Implicit (2-legged) Tutorial](./oauth-implicit.html)
+
+### Authorization Code (3-legged)
+
+* Safest one
+
+* Able to authenticate both app and user
+
+* Allows to obtain a long-lived access token  renewed with a refresh token
+
+* Access and refresh token never reach the end user (stays at the web server)
+
+### Client Credentials (2-legged)
+
+* Used when the client has been developed by the same authority as the authorization server
+
+* For scenarios in which authenticating the end user is not required.
+
+* Able to authenticate the app (no user authorization).
+
+### Implicit (2-legged)
+
+* For browser-based or mobile apps, typically running JavaScript
+
+* Able to authenticate the User only (not the app! any app could use the token => insecure)
+
+* Browser (or mobile app) receives access token, not the app server
+
+* No refresh token granted (because it's less secure)
+
+* Attention! This type of authorization should only be used if no other type of authorization is available. It is the least secure because the access token is exposed (and therefore vulnerable) on the client side.
+
+##  How do i use it?
+
+* Authorization Code: First, you need an authorization code ```c/oauth2/auth``` before you can call the get access token method. See the **[OAuth Authorization Code Tutorial](./oauth-authorisation-code.html)** and learn how to get an authorization code and how to use it.
+
+* Client Credetials: Call the access token method directly. No code is needed. It is important to use ```grant_type=client_credential```. Have look at the **[OAuth Client Credential Tutorial](./oauth-client-credentails.html)**.
+
+* Implicit: Use the ```c/oauth2/auth``` method like to get the authorization code, except the response_type has changed to response_type=token (instead of response_type=code). See the **[OAuth Implict Tutorial](./oauth-implicit.html)**  and learn how to use this method. The access token will be returned in the redirect URL directly.
+  
+![OAuth grand types description](img/oauth10.png)
+
+The POST method token with grant_type=refresh_token has to be used, if the time to live of the access token has been expired. 
+TTL (default):
+
+* access_token: 1209600000ms (14 days)
+
+* refresh_token: 31536000000ms (365 days)
+
+It is possible to get information about the access and refresh token by calling the GET method info.
+Sometimes an access token has to  be revoken. In this case use the GET method revoke.
+
+##  <a id='OAuth_API'></a> OAuth API documentation (https://consent.swisscom.com/)
+
+---
+
+### /c/oauth2
+
+#### /c/oauth2/auth
+Used to obtain the authorization codes (OAuth 3-legged) or the access token (implicit 2-legged)
+
+* **get**: Used to obtain the authorization codes or the access token (implicit)
```

Comment:
It wasn't the final version and this fragment was generated from raml2md generator.
I fixed it manually for the time beeing.